javascript-functions/main.js

```diff
@@ -0,0 +1,34 @@
+function convertMinutesToSeconds(minutes) {
+  var convert = minutes * 60 + ' seconds';
+  return convert;
+}
```

Comment:
`convertMinutesToSeconds` should return a number, not a string:

---

![image](https://user-images.githubusercontent.com/16143771/177617452-3be6e00e-f7d3-497b-8541-209b37c58f63.png)
****

Reply:
took out + 'minutes'